Website/Packages/datax-pipeline/src/modules/flowDefinition/flowSelectors.js

```diff
@@ -235,11 +240,17 @@ export const getEnableLocalOneBox = createSelector(
 // Validation - Info
 export const validateFlowInfo = createSelector(
     getFlowDisplayName,
+    getFlowDatabricksToken,
+    getFlowIsDatabricksSparkType,
     validateInfo
 );
 
-function validateInfo(displayName) {
-    return displayName && displayName.trim() !== '';
+function validateInfo(displayName, databricksToken, isDatabricksSparkType) {
+    if (isDatabricksSparkType) {
+        return displayName && displayName.trim() !== '' && databricksToken && databricksToken.startsWith('secretscope://');
```

Comment:
Let's create/use a constant for "secretscope://"

Reply:
Done